src/components/CommentView/CommentView.tsx

```diff
@@ -144,6 +144,7 @@ export const CommentView: FC<CommentViewProps> = ({
                 label: tr('Copy raw'),
                 icon: <IconClipboardOutline size="xxs" />,
                 onClick: () => notifyPromise(copyValue(descriptionRef.current), 'copy'),
+                className: null,
```

Comment:
Why explicitly indicate that there is no class?

Reply:
<img width="767" alt="Screenshot 2024-04-10 at 15 22 26" src="https://github.com/taskany-inc/issues/assets/9020793/e33dc82d-0a3e-4472-90b8-812e943ffe19">